docs/cloud/connections.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 3
+---
+
+# Peering Connections
+
+*Peering connections* are used  to connect a Dragonfly Cloud private network with a VPC in your cloud account to establish communication between the two networks over private ip space.
+
+Private IP space communication is more secure, performant and reduces data transfer costs incurred by cloud providers.
+
+Once you created a *private network* as described in [Networks](./networks) click the network three dots menu and click + Connect
+
+Continue to [AWS](#aws) or [GCP](#gcp) based on your cloud provider.
+
+
+## AWS
+Specify the *region*, *CIDR*, *account ID* (also called owner ID in AWS) and *VPC ID* of your AWS VPC from where you want to connect and click *Create*.
+
```

Comment:
Add explanation about hte CIDR. what it should be. How to choose?

Reply:
it says "of you AWS VPC" and you only get here after crating a network so should be familiary with CIDR already

we have a few ideas to imporve the experience around connection creation but don't want to block on that now